api/controller/users.js

```diff
@@ -559,11 +554,10 @@ function addQuery(filter, params, query, default_val) {
 
 	//check default case
 	query = query || {};
-
 	//validate
 	if (typeof params[filter] != "undefined" && typeof params[filter] === "string") {
-
-		if (filter == 'q') {
+		
+		if(filter=='username'){
```

Comment:
There is no point in replacing `q` with `username`. `q` is commonly used for search operations.